extensions/amp-bind/0.1/test/test-bind-integration.js

```diff
@@ -53,10 +52,14 @@ describe.configure().retryOnSaucelabs().run('integration amp-bind', function() {
     });
   }
 
+  function setButtonBinding(button, binding) {
+    button.setAttribute('on', `tap:AMP.setState(${binding})`);
```

Comment:
Bit of a misnomer. A "binding" is a (property, expression) tuple on an expression -- the element's property is "bound" to the expression. Updating a state var via `AMP.setState` is not a binding.

I'd rename this method `function setStateOnTap(button, key, value)`.